build/yamls/antrea-aks.yml

```diff
@@ -118,6 +118,19 @@ spec:
                         properties:
                           port:
                             x-kubernetes-int-or-string: true
+                          portRange:
+                            properties:
+                              except:
+                                items:
+                                  type: integer
+                                type: array
+                              from:
+                                type: integer
+                              port:
```

Comment:
Based on upstream discussions it seems the majority vote is against having the port field inside portRange https://github.com/kubernetes/enhancements/pull/2090#discussion_r518264979

Reply:
Cool that makes it clear. I will change it.